oaknational/Oak-Web-Application
# Feat/lesq 526/index ks pages [LESQ-526]

## Description

Music year: 1997

- create sitemap for keystage listing pages
- remove nofollow noindex from keystage subject pages
- there is a companion[ PR in Acorn](https://github.com/oaknational/Acorn/pull/765) for this ticket

## How to test

1. Go to https://deploy-preview-2193--oak-web-application.netlify.thenational.academy/teachers/key-stages/ks3/subjects
2. Inspect the page
3. You should see the robots meta tag is now `index,follow`
4. Go to https://deploy-preview-2193--oak-web-application.netlify.thenational.academy/teachers/key-stages/sitemap.xml
5. You should see the subject listing pages
